src/integration-tests/bash/cleanup.sh

```diff
@@ -77,282 +60,128 @@ RESULT_DIR="$RESULT_ROOT/acceptance_test_tmp"
 USER_PROJECTS_DIR="$RESULT_DIR/user-projects"
 TMP_DIR="$RESULT_DIR/cleanup_tmp"
 JOB_NAME="weblogic-command-job"
-DRY_RUN="false"
-[ "$1" = "-dryrun" ] && DRY_RUN="true"
-
-echo @@ `timestamp` Starting cleanup.
-script="${BASH_SOURCE[0]}"
-scriptDir="$( cd "$(dirname "${script}")" > /dev/null 2>&1 ; pwd -P)"
-source $PROJECT_ROOT/kubernetes/internal/utility.sh
 
-if [ ! "$1" = "" ] && [ ! "$1" = "-dryrun" ]; then
-  echo "@@ `timestamp` Usage: '$(basename $0) [-dryrun]'. Pass -dryrun to skip deletes."
-  exit 1  
-fi
 
 function fail {
-  echo @@ `timestamp` cleanup.sh: Error "$@"
+  echo @@ cleanup.sh: Error "$@"
   exit 1
 }
 
-# use for kubectl delete of a specific name, exits silently if nothing found via 'get'
-# usage: doDeleteByName [-n foobar] kind name
-function doDeleteByName {
-
-  local tmpfile="/tmp/$(basename $0).doDeleteByName.$PPID.$SECONDS"
-
-  kubectl get "$@" -o=jsonpath='{.items[*]}{.kind}{" "}{.metadata.name}{" -n "}{.metadata.namespace}{"\n"}' > $tmpfile
-
-  # exit silently if nothing to delete
-  [ `cat $tmpfile | wc -l` -eq 0 ] && return
+#!/bin/bash
```

Comment:
why a hashbang line?

Reply:
is this a bad merge?

Reply:
actually, it's in the model-in-image branch, but removing it now.